docs/kyma/docs/040-cr-installation.md

```diff
@@ -37,3 +49,7 @@ This table lists all the possible parameters of a given resource together with t
 | **metadata.finalizers** | **NO** | Protects the CR from deletion. Read [this](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#finalizers) Kubernetes document to learn more about finalizers. |
 | **spec.version** | **NO** | When manually installing Kyma on a cluster, specify any valid [SemVer](https://semver.org/) notation string. |
 | **spec.url** | **YES** | Specifies the location of the Kyma sources `tar.gz` package. For example, for the `master` branch of Kyma, the address is `https://github.com/kyma-project/kyma/archive/master.tar.gz` |
+| **spec.components** | **YES** | List of Helm charts components to `install` or `update` |
+| **spec.components.name** | **YES** | The name of the component is also the name of the component subdirectory in the `resources` directory. |
+| **spec.components.namespace** | **YES** | Namespace of the component in which it will be installed or updated. |
+| **spec.components.release** | **NO** | The name of release in case it should differ from component name. |
```

Comment:
I don't think it is obvious where I can take this value from, I assume it is helm release name